cypress/integration/student-competency-calculations.js

```diff
@@ -21,8 +21,8 @@ describe('Confirm rounding is consistent across UI, API, and exports', () => {
                     .should(({ xhr }) => {
                         const studentCompetencySuffixes = Object.keys(testCases[studentUsername][studentContentArea]);
```

Comment:
**nitpick** lets rename `studentCompetencySuffixes` since what it contains is no longer suffixes, they are now `studentCompetencyCodes`

It makes the rest of the code confusing to read

Reply:
done